components/cardTitle/CreateCountdownTitle.jsx

```diff
@@ -0,0 +1,42 @@
+import styled from "styled-components";
+
+export const CreateCountdownTitle = ({ onCreate }) => {
+  return (
+    <FormStyled
+      onSubmit={(event) => {
+        const newTitle = event.target.elements.newTitle.value; //event.target >> always the place where event take place!!
+        onCreate(newTitle);
+        event.preventDefault();
+      }}
+    >
+      <TextInputStyled
+        type="text"
+        name="newTitle"
+        placeholder="enter countdown name"
+      />
```

Comment:
consider adding a label for the input field here.

Reply:
check